style.css

```diff
@@ -400,7 +401,7 @@ h2 {
   width: 124px;
   height: 48px;
   left: 0;
```

Comment:
The "Left" property can only be used in conjunction with the "position: absolute" property.
Passing the "Left" property without the "position: absolute" property would create no effect.
It may be better to delete the "Left" property as it has no effect in the code.